docs/EN/Getting-Started/Screenshots.md

```diff
@@ -122,7 +122,11 @@ Here we see the status of the insulin pump - in this case an Accu-Chek Combo. Th
 
 ![Care Portal](../images/Screenshot_care_portal.png)
 
-This replicates the functions you will find on your Nightscout screen under the "+" symbol which allows you to add notes to your records. Functions such as recording when you change a pump site, or insulin cartridge should be self explanatory. BUT this section does not issue any commands to your pump. So if you add a bolus using this screen it simply makes a note of this on your Nightscout record, the pump won't be instructed to deliver a bolus. 
+The AAPS Careportal is only for entries in the database and not for actual treatments. It corresponds to the functions that you find on Nightscout under the "+" symbol. These allow you to add additional notes to your records. Functions like changing catheters and reservoirs should be self-explanatory. 
+
+ATTENTION: No commands can be sent to your pump from the AAPS-Care Portal! So if you add a bolus via the Care Portal, only a corresponding note will be entered in Nightscout. The pump itself will NOT deliver a bolus. The distinction between snack, meal, and correction boluses is rather arbitrary, and the input dialogue is always the same. Possible uses: snack = snack between meals, meal = meal, correction = hypo-carbohydrates.
```

Comment:
'is rather arbitrary, and the input dialogue is always the same.' please delete that because it is your opinion, no fact

Reply:
![Screenshot_20200102-005323_AndroidAPS](https://user-images.githubusercontent.com/46249367/71647716-29ec8000-2cfb-11ea-8a67-4cc1f3d31221.jpg)
![Screenshot_20200102-005315_AndroidAPS](https://user-images.githubusercontent.com/46249367/71647717-2eb13400-2cfb-11ea-9b3a-80543b8c0ea8.jpg)
![Screenshot_20200102-005303_AndroidAPS](https://user-images.githubusercontent.com/46249367/71647719-3244bb00-2cfb-11ea-90c8-1af1ba29c659.jpg)
Hm. As far as I can see, they ARE the same except the title. I am rather new to AAPS, so I asked the German loop community, all agreeing. 
Could you please tell me the difference? How should these buttons be used?
Thanks!